plugins/network_vis/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+npm-debug.log
```

Comment:
I am thinking about how we could make this plugin more independent than checking all the code into github (thinking about the Kibana upgrade process in the future)

We might want to actually just host the plugin zip file in the artifactory, and untar it as part of the build process. That'd make upgrading it and kibana each more flexible. What do you think?

Reply:
I think we could even take care of the `material-ui` pathing change from the spec file with a `sed` command as well

Reply:
Discussed IRL - moved the plugin zip file to `kibana/resources/plugins` and unzip/install it during the build. Not implementing sed commands for `material-ui` since we only needed the changes for our own files.